trl/trainer/online_dpo_trainer.py

```diff
@@ -371,6 +352,13 @@ def get_eval_dataloader(self, eval_dataset: Optional[Union[str, Dataset]] = None
     def training_step(self, model: nn.Module, inputs: Dict[str, Union[torch.Tensor, Any]]) -> torch.Tensor:
         model.train()
 
+        # Apply chat template and tokenize the input
```

Comment:
Would this make sense to include here?

```suggestion
        # Apply chat template and tokenize the input.
        # We do this on-the-fly to enable the use of reward models and policies with different tokenizers / chat templates.
```